tests/cstor/volume/provision_bulk_test.go

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import (
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	sc "github.com/openebs/maya/pkg/kubernetes/storageclass/v1alpha1"
+	spc "github.com/openebs/maya/pkg/storagepoolclaim/v1alpha1"
+	"github.com/openebs/maya/tests/cstor"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("[cstor] TEST BULK VOLUME PROVISIONING", func() {
+	var (
+		err           error
+		pvcNamePrefix = "cstor-volume-claim-"
+		bulkCount     = 10
+		pvcObjList    *v1.PersistentVolumeClaimList
+		pvcObjTpl     *v1.PersistentVolumeClaim
+	)
+
+	When("creating resources required by test", func() {
+		It("should create a spc and storageclass", func() {
+
+			By("building spc")
+			spcObj = spc.NewBuilder().
+				WithGenerateName(spcName).
+				WithDiskType(string(apis.TypeSparseCPV)).
```

Comment:
Can we make use of apis type?
Let us not use string as argument

Reply:
Yes we can. I will raise a separate PR as this will require refactoring at many places.